test/extended/apiserver/resiliency.go

```diff
@@ -52,6 +57,13 @@ var _ = ginkgo.Describe("[Conformance][sig-sno][Serial] Cluster", func() {
 		framework.ExpectNoError(err)
 		gomega.Expect(clusterApiServer.Status.NodeStatuses[0].TargetRevision).To(gomega.Equal(int32(0)))
 
+		ginkgo.By("Checking pods restart count")
+		restartingContainers := make(map[operators.ContainerName]int)
+		c, err := e2e.LoadClientset()
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		// This will just load the restartingContainers map with the current restart count
+		_ = GetRestartedPods(c, restartingContainers)
```

Comment:
Why this code block? It needs a comment. It's not clear to me why we should care about pod restart counts.

Reply:
Updated the comment